packages/angular_devkit/core/src/workspace/json/reader.ts

```diff
@@ -60,8 +60,10 @@ export async function readJsonWorkspace(
       // TODO: Diagnostic reporting support
       throw new Error(message);
     },
-    warn(_message, _node) {
+    warn(message, _node) {
       // TODO: Diagnostic reporting support
+      // eslint-disable-next-line no-console
+      console.warn(message);
```

Comment:
nit: Does the node provide location info? line/column info may be useful for fixing the underlying problem.